terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1017,6 +1017,15 @@ def qq(*words):
             }
             for zone in range(num_zones)
         },
+        'aws_vpc_endpoint': {
+            f'{service}_gateway': {
+                'service_name': f'com.amazonaws.{config.region}.{service}',
+                'vpc_endpoint_type': 'Gateway',
+                'vpc_id': '${aws_vpc.gitlab.id}',
+                'route_table_ids': [f'${{aws_route_table.gitlab_{i}.id}}' for i in range(num_zones)],
```

Comment:
I believe we prefer using `%` for string interpolation in conjunction with Terraform's `{}` syntax

```suggestion
                'route_table_ids': ['${aws_route_table.gitlab_%d.id}' % i for i in range(num_zones)],
```